aboutus.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="DRRP Website">
+
+		<title>Transition to Employment</title>
+
+		<!--- google fonts -->
+		<link href="https://fonts.googleapis.com/css?family=Fjalla+One" rel="stylesheet"> 
+
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+		
+	</head>
+	
+	<body>
+		<div class="container">
+		<header class="clearfix">
+			<div class="logo">
+				<a href="index.html"><img src="images/logo.png" alt="DRRP Transition to Employment logo"></a>
+			</div>
+				<nav>
+					<ul>
+						<li><a href="aboutus.html">ABOUT US</a>
```

Comment:
You can probably change the text to be `About Us` and then use CSS to make it caps:

```css
text-transform: uppercase;
```